src/App.js

```diff
@@ -1,7 +1,34 @@
 import React from 'react';
 import './App.css';
 
+import { useQuery } from '@apollo/react-hooks';
+import gql from 'graphql-tag';
+
+//sample data being pulled from pokemon api
+const GET_POKEMON_INFO = gql`
```

Comment:
Can you create a folder inside the `src` directory called `graphql`, and inside the `graphql` directory, create a `queries.js` and `mutations.js` files. Add the and export the `query` into the `queries.js` file, so we don't have queries and mutations scattered across components.